packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -128,6 +169,43 @@ export class AzureClient {
         return { container: fluidContainer, services };
     }
 
+    /**
+     * Get the list of versions for specific container
+     * @param id - Unique ID of the source container in Azure Fluid Relay.
+     * @param maxCount - Max number of versions to retreive,
+     * @returns Array of available versions
+     */
+     public async getContainerVersions(
+        id: string,
+        maxCount: number,
+    ): Promise<AzureContainerVersion[]> {
+        const url = new URL(this.props.connection.orderer);
+        url.searchParams.append(
+            "storage",
+            encodeURIComponent(this.props.connection.storage),
+        );
+        url.searchParams.append(
+            "tenantId",
+            encodeURIComponent(this.props.connection.tenantId),
+        );
+        url.searchParams.append("containerId", encodeURIComponent(id));
+
+        const resolvedUrl = await this.urlResolver.resolve({ url: url.href });
+        if (!resolvedUrl) {
+            throw new Error("Unable to resolved URL");
+        }
+        const documentService =
+            await this.documentServiceFactory.createDocumentService(
+                resolvedUrl,
+            );
+        const storage = await documentService.connectToStorage();
+        const versions = await storage.getVersions(null, maxCount);
```

Comment:
nit: can this be undefined?  i know that area of the code uses a lot of nulls but we've tried to move away from that everywhere else

Reply:
don't we have some no-null linting rule?